openra/classes/file_location.py

```diff
@@ -18,6 +20,16 @@ def __init__(self, fs: FS, path: str, file: str):
         self.path = path
         self.file = file
 
+    def get_file_basename(self):
+        return re.sub(r"\.[^\.]+$", "", self.file)
+
+    def get_file_extension(self):
+        search = re.search(r"\.([^\.]+)$", self.file)
```

Comment:
`os.path.splitext`?

Reply:
I'm just testing this now, and it's returning the '.' part as well, which isn't really needed

Reply:
This is updated now as above